pitch/auto-transpose/README.md

```diff
@@ -1,15 +1,14 @@
 The auto-transpose engraver can be used to automatically transpose music according to the instrument transposition. The engraver has to know, if the music it finds in this context is in concert- or in instrument-pitch. Then it can use the `instrumentTransposition`, which is set by the `\transposition` command, to transpose the music as necessary.
 
-The engraver looks for three context-properties:
+The engraver looks for two context-properties:
 1. instrumentTransposition
-2. music-concert-pitch
-3. print-concert-pitch
+2. transposeDirection
 
-The first one is a standard lilypond-context-property, which is used to set instrument transposition for MIDI-output. So, if you use for example `\transposition bes` -- e.g. trumpet or clarinet -- MIDI-output will sound one note lower. Now the other two context-properties are used to set, if the music is provided in concert-pitch and if it shall be printed in concert-pitch. If the music is given in concert pitch and shall be printed in instrument pitch, the engraver transposes it accordingly. `\autoTranspose` provides a context-mod which consists the engraver and sets the *-concert-pitch variables to display concert-pitched music in instrument pitch.
+The first one is a standard lilypond-context-property, which is used to set instrument transposition for MIDI-output. So, if you use for example `\transposition bes` -- e.g. trumpet or clarinet -- MIDI-output will sound one note lower. The other context-property is used to set if the music is provided in concert-pitch and printed in instrument pitch, or vica versa. If the music is given in concert pitch and shall be printed in instrument pitch, the engraver transposes it accordingly. `\autoTranspose` provides a context-mod which consists the engraver and sets transposeDirection to display concert-pitched music in instrument pitch.
```

Comment:
This paragraph could be clearer (I know it was not well-written to start with). Can you rewrite from scratch please including examples of actually setting these context-properties. Don't use "the first one"/"the other one" - just restate the name of the property.

Reply:
Rewrote the Readme - let me know what you think